app/lib/receipt.rb

```diff
@@ -0,0 +1,28 @@
+module Receipt
+  def print_receipt_head(shop_details)
```

Comment:
* This should not be a module
* It's worth taking a look with fresh eyes for a refactor. It's not very clear what's going on. Try and break everything into bitesize pieces.